tests/puppeteer/pages/BO/advancedParameters/team/index.js

```diff
@@ -22,7 +22,7 @@ module.exports = class Employees extends BOBasePage {
     this.employeesListTableToggleDropDown = row => `${this.employeesListTableColumnAction(row)
     } a[data-toggle='dropdown']`;
     this.employeesListTableDeleteLink = row => `${this.employeesListTableColumnAction(row)} a`
-      + `[data-confirm-button-label='Delete']`;
+      + '[data-confirm-button-label=\'Delete\']';
```

Comment:
Is this necessary ?

Reply:
eslint replace automatically ` with ' in strings when you don't have var in it

Reply:
OK. Damn ESLint.